windows/src/ext/sentry/sentry.h

```diff
@@ -657,7 +831,53 @@ SENTRY_API void sentry_options_set_debug(sentry_options_t *opts, int debug);
 SENTRY_API int sentry_options_get_debug(const sentry_options_t *opts);
 
 /**
- * Enables or disabled user consent requirements for uploads.
+ * Sets the number of breadcrumbs being tracked and attached to events.
+ *
+ * Defaults to 100.
+ */
+SENTRY_API void sentry_options_set_max_breadcrumbs(
+    sentry_options_t *opts, size_t max_breadcrumbs);
+
+/**
+ * Gets the number of breadcrumbs being tracked and attached to events.
+ */
+SENTRY_API size_t sentry_options_get_max_breadcrumbs(
+    const sentry_options_t *opts);
+
+/**
+ * Type of the callback for logger function.
+ */
+typedef void (*sentry_logger_function_t)(
+    sentry_level_t level, const char *message, va_list args, void *userdata);
+
+/**
+ * Sets the sentry-native logger function.
+ *
+ * Used for logging debug events when the `debug` option is set to true.
+ */
+SENTRY_API void sentry_options_set_logger(
+    sentry_options_t *opts, sentry_logger_function_t func, void *userdata);
+
+/**
+ * Enables or disables automatic session tracking.
+ *
+ * Automatic session tracking is enabled by default and is equivalent to calling
+ * `sentry_start_session` after startup.
+ * There can only be one running session, and the current session will always be
+ * closed implicitly by `sentry_close`, when starting a new session with
+ * `sentry_start_session`, or manually by calling `sentry_end_session`.
+ */
+SENTRY_API void sentry_options_set_auto_session_tracking(
+    sentry_options_t *opts, int val);
+
+/**
+ * Returns true if automatic session tracking is enabled.
```

Comment:
I noticed l.78 includes stdarg.h instead of stdbool.h. Does it mean this returns `1` if automatic session tracking is enabled?

Reply:
I guess so? I would take it as returning non-zero meaning true. This file is straight from Sentry, so I've placed it here unchanged.